src/renderer/views/ProjectsList/subcomponents/ProjectsTable/ProjectsTable.tsx

```diff
@@ -30,19 +34,30 @@ const ProjectTable: React.FC<Props> = ({ publications }) => {
     );
   };
 
+  const activePublicationID: string | undefined =
+    useSelector(activePublication)?.id;
+
   return (
     <Table sx={{ position: 'relative', zIndex: 1 }}>
       <TableHeader />
 
       {publications.map((publication) => {
-        const isSelected = selectedProject?.id === publication.id;
+        const isDescriptionVisible = selectedProject?.id === publication.id;
+        const isActive = activePublicationID === publication.id;
         return (
           <TableBody key={publication.id}>
-            <ProjectRow publication={publication} isSelected={isSelected} />
-            <SideviewIndicatorRow
-              isSelected={isSelected}
-              onClick={() =>
-                selectProject(isSelected ? undefined : publication)
+            <ProjectRow
+              publication={publication}
+              isSelected={isDescriptionVisible}
+            />
+            <ButtonRow
+              isDescriptionVisible={isDescriptionVisible}
+              onClickDescription={() =>
+                selectProject(isDescriptionVisible ? undefined : publication)
+              }
+              isProjectActive={isActive}
+              onClickActivePublication={() =>
+                dispatch(setActivePublication(publication.id))
               }
```

Comment:
Good question. I'm slightly worried that it would be too abrupt - the button looks different (and less significant) from any other button triggering the view change. I was thinking about changing the button under the active project to one that changes the view (instead of simple "active" text), but this is something we should brainstorm when thinking of the final look, along with other visual prompts. 

For now I have implemented the view change. One way or another, it is easier to get feedback on abrupt interaction then on lack of one.

Reply:
@LostInStatic yeah I see your point. We can gather some feedback on release and see from there. Easy enough to change this behavior anyway. 👍